plugins/catalog-backend/src/next/NextEntitiesCatalog.ts

```diff
@@ -80,7 +82,7 @@ export class NextEntitiesCatalog implements EntitiesCatalog {
           // NOTE(freben): This used to be a set of OUTER JOIN, which may seem to
           // make a lot of sense. However, it had abysmal performance on sqlite
           // when datasets grew large, so we're using IN instead.
-          const matchQuery = db<DbEntitiesSearchRow>('search')
+          const matchQuery = db<DbSearchRow>('search')
```

Comment:
whoops, was using the old type here before (but the right table - which actually worked out anyway in practice)